booklets_text/2014.txt

```diff
@@ -681,6 +682,7 @@ can create a new account, log in with authorisation and send information to each
 a ‘message’.
 The project is a proof of concept to prove that the underline technologies are there for an application
 to be implemented in the real world with off the shelf technology.
+Project Area: 
```

Comment:
Add missing area in a couple of places so that description can be extracted. Area will be empty string "" in these cases.